### DISCUSSION TITLE
Protocol handler for magnet links and torrent files

### DISCUSSION THREAD

**Discussion by jdarias** [jesec/flood#304] [Ideas]
Is it possible to register flood to handle magnet: protocol in my browser?

**Comment by jesec** [comment]
Yes. Flood has the feature since 4.4: https://flood.js.org/Changelog-4.4 . 

![image](https://user-images.githubusercontent.com/11585141/118286841-76f28000-b505-11eb-954d-ef7e28644c88.png)

**Reply by brezinajn** [reply]
Are there any required steps? The link to register seems to be missing. (v 4.5.1)
![image](https://user-images.githubusercontent.com/15677736/118291702-2b75ad00-b4d8-11eb-9d3a-2bc8facfa026.png)

**Reply by jesec** [reply]
If it is missing, you browser may not support it.

**Reply by jesec** [reply]
Note that the feature is only available in Secure Context (HTTPS) with newer browsers: 

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler

**Reply by brezinajn** [reply]
Oh, HTTPS is the culprit. Thx
